src/lib/action.rs

```diff
@@ -59,6 +59,10 @@ pub enum ActionType {
     MOVE,
     /// Update the text or label associated with a node.
     UPDATE,
+    /// Copy the text or label associated with a node or subtree.
```

Comment:
I think this copies the whole nodes / subtrees, right? Not just the labels and values (this is maybe a little pedantic, because the paper suggests that nodes contain a little less information than the Diffract implementation puts inside them)